packages/server/lib/errors.coffee

```diff
@@ -721,6 +721,12 @@ getMsgByType = (type, arg1 = {}, arg2) ->
 
       #{arg1.link}
       """
+    when "FIXTURE_NOT_FOUND"
+      """
+        A fixture with a supported file extension does not exist at the supplied path:
```

Comment:
I'm confused by this message and what actionable thing there is for me to do here. 

- What is Cypress currently interpreting as the file extension from my fixture command?
- Do I need to provide the file extension? What do I do to fix this!
- What file extensions *are* supported?

All of these questions should be answered in the error message. If needed, link to a on.cypress.io link.

Reply:
Good point, it's not very clear. I updated the error message with details on what was tried and what they should do when we can't find their fixture:

![image](https://user-images.githubusercontent.com/1151760/53521601-975dcd80-3aa6-11e9-8a5f-1097a9c6cd7f.png)

Reply:
Is this awful? 

```
Error: A fixture file could not be found at any of the following paths:

 > cypress/fixtures/err.txt
 > cypress/fixtures/err.txt{{extension}}

Supported fixture file extensions include: 

 > .json, .js, .coffee, .html, .txt, .csv, .png, .jpg, .jpeg, .gif, .tif, .tiff, .zip

Provide a path to an existing fixture file.
```

Reply:
It's a little misleading, since technically "any" file extension is supported, but those are just the extensions we attempt to tack on to the filename if the file can't be found at the user-supplied filename. I reworded it a bit:

![image](https://user-images.githubusercontent.com/1151760/53524124-9af45300-3aac-11e9-902a-5397d30860d6.png)

?

Reply:
Maybe 

"Cypress looked for these file extensions at the provided path"

Reply:
Sounds good, changed it to that